### DISCUSSION TITLE
离线设置无效

### DISCUSSION THREAD

**Discussion by bfqbpq567** [shuax/MouseInc.Settings#473] [General]
1，下载zip后，在mouseinc文件夹，也下载webview2。一开始能用本地，但过一段时间打开设置，又变成联网的了，在浏览器打开。发现重新安装一下好了，但过一段时间，再打开又是联网浏览器打开
2，现在尝试另一种下载node，还是没用。
![image_67](https://user-images.githubusercontent.com/66172873/205489415-ee13138e-e740-4d5e-a537-d36588de3532.png)
![image_68](https://user-images.githubusercontent.com/66172873/205489421-d129ac49-1bc6-4172-8ed2-80c5798037b1.png)
![image_70](https://user-images.githubusercontent.com/66172873/205489422-72ec7736-7108-454c-9b35-24cdc498c7cc.png)

**Comment by bfqbpq567** [comment]
重新覆盖安装一下webview2又可以离线设置了，只是不知道到时又会不会出问题了？这是为啥(⊙o⊙)…

**Comment by bfqbpq567** [comment]
目前离线设置没问题。(*^▽^*)打扰shuax大大了

**Comment by MTFAlpha1** [comment]
能请教一下如何设置离线吗？

**Reply by bfqbpq567** [reply]
![image_72](https://user-images.githubusercontent.com/66172873/208133661-6e3c7bcb-f75b-40eb-b872-b30bf9d9e9ff.png)
下载webview2这个软件安装和离线设置压缩包就行了

**Reply by bfqbpq567** [reply]
https://github.com/shuax/MouseInc.Settings/releases/tag/2.13.2
